packages/device-connector/src/util/common.ts

```diff
@@ -0,0 +1,26 @@
+export enum grpcStatus {
```

Comment:
prefer capitalized enums

Reply:
resolved in 03c659a4b443a8c48d6aeaddcf6f30e1de3a948d